package.json

```diff
@@ -44,7 +44,7 @@
         "remark-rehype": "^7.0.0",
         "styled-components": "^4.3.2",
         "unified": "^9.0.0",
-        "xlsx": "^0.14.4"
+        "xlsx": "^0.17.0"
```

Comment:
@TahimiLeonBravo we need to test this out since it is an upgrade in a direct dependency, the ImportRecordsFlow component is the one that uses this dependency then we must make some QA before merging